### PR TITLE
[libc++][z/OS] Fixup two linear_congruential_engine tests

### PR DESCRIPTION
**Pull request by zibi2**

### CODE REVIEW

**Review by muiez — APPROVED** [review]
LGTM

File: libcxx/test/std/numerics/rand/rand.eng/rand.eng.lcong/assign.pass.cpp
Comment (mordante) [inline libcxx/test/std/numerics/rand/rand.eng/rand.eng.lcong/assign.pass.cpp:87]: This is wrong. The code should test for `TEST_HAS_NO_INT128`.

**Review by mordante — COMMENTED** [review]
Next time, please wait for code owners to have a chance to review your changes.

Please provide a followup patch to fix this in the proper way.

File: libcxx/test/std/numerics/rand/rand.eng/rand.eng.lcong/assign.pass.cpp
Comment (zibi2) [reply libcxx/test/std/numerics/rand/rand.eng/rand.eng.lcong/assign.pass.cpp:87]: > This is wrong. The code should test for `TEST_HAS_NO_INT128`.

Sure, I will do this for existing guards as well.

**Comment by ldionne** [comment]
@zibi2 Please disable the "hide my email" feature in Github. Otherwise, your commits are being committed under `62662650+zibi2@users.noreply.github.com` and we require a valid email address for every commit.

**Comment by zibi2** [comment]
> @zibi2 Please disable the "hide my email" feature in Github. Otherwise, your commits are being committed under `62662650+zibi2@users.noreply.github.com` and we require a valid email address for every commit.

Yes, I willl.

**Pull request merged**
